android/src/com/mapswithme/maps/routing/NavigationController.java

```diff
@@ -103,6 +161,26 @@ public void onClick(View v)
     mDistanceValue = (TextView) mBottomFrame.findViewById(R.id.distance_value);
     mDistanceUnits = (TextView) mBottomFrame.findViewById(R.id.distance_dimen);
     mRouteProgress = (FlatProgressView) mBottomFrame.findViewById(R.id.navigation_progress);
+
+    // Search
+    mSearchButton = (ImageView) mFrame.findViewById(R.id.btn_search);
+    mSearchButton.setOnClickListener(this);
+    mSearchLayout = mFrame.findViewById(R.id.search_frame);
+    if (UiUtils.isLandscape(mFrame.getContext()))
+    {
+      UiUtils.waitLayout(mSearchLayout, new ViewTreeObserver.OnGlobalLayoutListener()
+      {
+        @Override
+        public void onGlobalLayout()
+        {
+          mSearchLayout.setPivotX(0);
+          mSearchLayout.setPivotX(mSearchLayout.getMeasuredHeight() / 2);
```

Comment:
Good catch, second one should be setPivotY, actually. Thx!

> On Sep 9, 2016, at 1:55 PM, ygorshenin <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/routing/NavigationController.java <https://github.com/mapsme/omim/pull/4246#discussion_r78162059>:
> 
> > @@ -103,6 +161,26 @@ public void onClick(View v)
> >      mDistanceValue = (TextView) mBottomFrame.findViewById(R.id.distance_value);
> >      mDistanceUnits = (TextView) mBottomFrame.findViewById(R.id.distance_dimen);
> >      mRouteProgress = (FlatProgressView) mBottomFrame.findViewById(R.id.navigation_progress);
> > +
> > +    // Search
> > +    mSearchButton = (ImageView) mFrame.findViewById(R.id.btn_search);
> > +    mSearchButton.setOnClickListener(this);
> > +    mSearchLayout = mFrame.findViewById(R.id.search_frame);
> > +    if (UiUtils.isLandscape(mFrame.getContext()))
> > +    {
> > +      UiUtils.waitLayout(mSearchLayout, new ViewTreeObserver.OnGlobalLayoutListener()
> > +      {
> > +        @Override
> > +        public void onGlobalLayout()
> > +        {
> > +          mSearchLayout.setPivotX(0);
> > +          mSearchLayout.setPivotX(mSearchLayout.getMeasuredHeight() / 2);
> Could you please explain what's the point to call setPivotX two times?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/mapsme/omim/pull/4246/files/f104f4709dfa59a30e3d90cfa6d858d0b51ba4ec#r78162059>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ACKTRIcf5fhjBLpYj6F7d5wfnTqecljjks5qoTstgaJpZM4J4yII>.
>